src/main/java/in/ashwanthkumar/gocd/slack/GoNotificationMessage.java

```diff
@@ -188,7 +195,7 @@ public void tryToFixStageResult(Rules rules)
                                              Integer.parseInt(pipeline.stage.counter));
         if (previous == null || StringUtils.isEmpty(previous.result)) {
             LOG.info("Couldn't find any previous run of " +
-                     pipeline.name + "/" + pipeline.counter + "/" +
+                     pipeline.name + "/" + pipeline.label + "/" + pipeline.counter + "/" +
```

Comment:
Having the label in this format is a breaking change because in GoCD it is the practice to have the pipeline counter after the name. Please consider having the label inside a `( ... label ... )` after the pipeline name to make it call out differently.

Reply:
Changes done

Reply:
@ashwanthkumar Kindly re-review PR.